consultation_analyser/consultations/views/consultations.py

```diff
@@ -31,6 +34,9 @@ def show(request: HttpRequest, consultation_slug: str) -> HttpResponse:
 
     context = {"questions": questions, "consultation": consultation}
 
+    if not consultation.has_themes():
```

Comment:
I think we deleted this method!

Reply:
Yes - I think this whole PR needs to be rewritten! (I started it before the other one was finished.)